atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -69,7 +76,9 @@ def __init__(self, tasker, workflow,
                  koji_krb_principal=None,
                  koji_krb_keytab=None,
                  to_koji_submitter=False,
-                 to_koji_pkgowner=False):
+                 to_koji_pkgowner=False,
+                 use_auth=None,
```

Comment:
Let's add a corresponding PR in osbs-client to set these new parameters.

Note: it's fine if osbs-client sets a plugin parameter that the plugin is not aware of. Atomic reactor is smart enough to discard those.